src/index.ts

```diff
@@ -79,29 +80,24 @@ export const createInstance = (config: Config): Instance => {
       return userClaim;
     },
     getSaleMintlistDuration: async (): Promise<ethers.BigNumber> => {
-      const contract = await getWolfSaleContract(
+      const contract = await getAirWild2SaleContract(
         config.web3Provider,
         config.contractAddress
       );
-      const duration = await contract.privateSaleDuration();
+      const duration = await contract.mintlistDurations(PRIVATE_SALE_INDEX);
```

Comment:
Why is this hardcoded to always be the private sale? How does the dApp get how long the public sale is?